asv_bench/benchmarks/index_object.py

```diff
@@ -138,7 +138,8 @@ def setup(self, dtype):
         self.sorted = self.idx.sort_values()
         half = N // 2
         self.non_unique = self.idx[:half].append(self.idx[:half])
-        self.non_unique_sorted = self.sorted[:half].append(self.sorted[:half])
+        self.non_unique_sorted = self.sorted[:half].append(self.sorted[:half])\
```

Comment:
Can you wrap the statement in parenthesis instead of using `\`. We would like to move away from them in the codebase

Reply:
Done